js/chocolatey-packages.js

```diff
@@ -137,35 +137,35 @@
                 default:
                     langBrush = "none";
             }
-            $(this).find('pre').addClass('line-numbers').find("code").addClass('language-' + langBrush);
-            Prism.highlightElement($(this).find('code')[0]);
+            jQuery(this).find('pre').addClass('line-numbers').find("code").addClass('language-' + langBrush);
+            Prism.highlightElement(jQuery(this).find('code')[0]);
         }
     });
     // Expand or Show all files
-    $('#files .btn-collapse-files').click(function () {
-        var $this = $(this);
-        var thisText = $this.text();
-        if ($this.hasClass('btn-success')) {
-            $this.text(thisText.replace('Expand', 'Collapse'));
-            $this.removeClass('btn-success').addClass('btn-danger');
-            $('#files .btn:contains("Show")').html('Hide');
+    jQuery('#files .btn-collapse-files').click(function () {
+        var jQuerythis = jQuery(this);
+        var thisText = jQuerythis.text();
+        if (jQuerythis.hasClass('btn-success')) {
+            jQuerythis.text(thisText.replace('Expand', 'Collapse'));
+            jQuerythis.removeClass('btn-success').addClass('btn-danger');
+            jQuery('#files .btn:contains("Show")').html('Hide');
             fileCollapse.collapse('show');
-        } else if ($this.hasClass('btn-danger')) {
-            $this.text(thisText.replace('Collapse', 'Expand'));
-            $this.removeClass('btn-danger').addClass('btn-success');
-            $('#files .btn:contains("Hide")').html('Show');
+        } else if (jQuerythis.hasClass('btn-danger')) {

```

Comment:
This doesn't look right to me.

Reply:
I have updated this snippet to use better variable names to make more sense.